SPECS/influxdb2/influxdb2.spec

```diff
@@ -0,0 +1,208 @@
+#
+
+Summary:        Scalable datastore for metrics, events, and real-time analytics
+Name:           influxdb2
+Version:        2.4.0
+Release:        1%{?dist}
+License:        MIT
+Vendor:         Microsoft Corporation
+Distribution:   Mariner
+Group:          Productivity/Databases/Servers
+URL:            https://github.com/influxdata/influxdb
+Source0:        %{url}/archive/refs/tags/v%{version}.tar.gz#/%{name}-%{version}.tar.gz
+# Below is a manually created tarball, no download link.
+# We're using pre-populated Go modules from this tarball, since network is disabled during build time.
+# How to re-build this file:
+#   1. wget https://github.com/influxdata/influxdb/archive/refs/tags/v%%{version}.tar.gz -O %%{name}-%%{version}.tar.gz
+#   2. tar -xf %%{name}-%%{version}.tar.gz
+#   3. cd %%{name}-%%{version}
+#   4. go mod vendor
+#   5. tar  --sort=name \
+#           --mtime="2021-04-26 00:00Z" \
+#           --owner=0 --group=0 --numeric-owner \
+#           --pax-option=exthdr.name=%d/PaxHeaders/%f,delete=atime,delete=ctime \
+#           -cf %%{name}-%%{version}-vendor.tar.gz vendor
+#
+Source1:        %{name}-%{version}-vendor.tar.gz
+# Below is a manually created tarball, no download link.
+# predownloaded assets include ui assets and swager json. Used to replace fetch-assets and fetch-swagger script.
+# How to rebuild this file:
+#   1. wget https://github.com/influxdata/influxdb/archive/refs/tags/v%%{version}.tar.gz -O %%{name}-%%{version}.tar.gz
+#   2. tar -xf %%{name}-%%{version}.tar.gz
+#   3. cd %%{name}-%%{version}
+#   4. make generate-web-assets
+#   5. cd static
+#   6. tar -cvf %%{name}-%%{version}-static-data.tar.gz data/
+Source2:        %{name}-%{version}-static-data.tar.gz
+Source3:        influxdb.service
+Source4:        influxdb.tmpfiles
+Source5:        config.yaml
+BuildRequires:  go >= 1.18
+BuildRequires:  golang-packaging >= 15.0.8
+BuildRequires:  pkgconfig(flux) >= 0.179.0
+BuildRequires:  protobuf-devel
+BuildRequires:  kernel-headers
+BuildRequires:  rust >= 1.60.0
+BuildRequires:  clang
+BuildRequires:  tzdata
+BuildRequires:  systemd-rpm-macros
+Requires:       tzdata
+Conflicts:      influxdb
+%{!?_tmpfilesdir:%global _tmpfilesdir /usr/lib/tmpfiles.d}
+%{?systemd_requires}
+Requires(post): systemd
+
+%description
+InfluxDB is an distributed time series database with no external dependencies.
+It's useful for recording metrics, events, and performing analytics.
+
+%package        devel
+Summary:        InfluxDB development files
+Group:          Development/Languages/Golang
+Requires:       go
+Requires:       tzdata
+Conflicts:      influxdb
+
+%description devel
+Go sources and other development files for InfluxDB
+
+%prep
+%autosetup
+
+%build
+export GOPATH=$HOME/go
+export GOBIN=$GOPATH/bin
+export PATH=$PATH:$GOPATH:$GOBIN
+export GO111MODULE=on
+tar -xf %{SOURCE1} --no-same-owner
+
+mkdir -pv static
+tar -xf %{SOURCE2} -C static/ --no-same-owner
```

Comment:
Nit-pick. Since the main source is extracted as part of the `%prep` step, I'd move these two extraction there as well. Also, I'd try to make use of the `-a` switch for `%autosetup` to do most of the work for us, if possible.